composer.json

```diff
@@ -24,21 +24,21 @@
         "filament/filament": "^3.2.23",
         "filament/spatie-laravel-settings-plugin": "^3.2",
         "guzzlehttp/guzzle": "^7.8",
-        "illuminate/console": "^10.0",
-        "illuminate/database": "^10.0",
-        "illuminate/events": "^10.0",
-        "illuminate/queue": "^10.0",
-        "illuminate/support": "^10.0",
+        "illuminate/console": "^11.0",
+        "illuminate/database": "^11.0",
+        "illuminate/events": "^11.0",
+        "illuminate/queue": "^11.0",
+        "illuminate/support": "^11.0",
         "nesbot/carbon": "^2.70",
```

Comment:
Laravel 11 also supports Carbon 3 but I don't know if you want to cover that right now. 

Some additional breaking changes might make it worth a separate PR.

Reply:
@crynobone thanks! Yeh I'll send in a follow up to isolate any BC problems there. 👍